homework/style.css

```diff
@@ -1,3 +1,50 @@
+*,
+*:before,
+*:after {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    list-style: none;
+    text-decoration: none;
+}
+
 .alert-error {
     color: red;
-}
\ No newline at end of file
+}
+
+#header-div{
+  background-color:#3F51B5;
+  height: 100px;
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+}
+
+h1 {
+  color: white;
+  margin: 0 20px 0 20px;
+}
+
+.container {
+  display: flex;
```

Comment:
Nice use of `flex`! But using media queries you could make it even more responsive!